cookbook/how_to_accelerate_import.rst

```diff
@@ -176,20 +183,15 @@ The following example shows services that should be created to add a new import
 
     # Custom strategy
     orocrm_contact.importexport.strategy.contact.add:
-        class: "%orocrm_contact.importexport.strategy.contact.add.class%"
+        class: %orocrm_contact.importexport.strategy.contact.add.class%
```

Comment:
I would be the quotes here. They are not required in your projects, but Sphinx requires them to render YAML properly (Sphinx internally uses a YAML 1.1 lexer).